### DISCUSSION TITLE
login.microsoftonline.com refused to connect with SPFx

### DISCUSSION THREAD

**Discussion by MAP-Git** [community/community#66028] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Facing issue: 'login.microsoftonline.com refused to connect'. It faced while deploy .sppkg file to Site. Solution works fine with workbench, but not works when deploy.

Please suggest solution for this problem.

Thanks in advance!

![SPFx Connection Refuse](https://github.com/community/community/assets/66069197/f751015e-df21-4a47-b381-a60a87b57afb)

**Comment by queenofcorgis** [comment]
Hi @MAP-Git,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Microsoft products like their [Tech Community](https://techcommunity.microsoft.com/t5/sharepoint/ct-p/SharePoint). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Reply by MAP-Git** [reply]
Thanks for letting me aware! I will post a question at Tech Community instead.
